plugin/trino-hudi/src/test/java/io/trino/plugin/hudi/TestHudiSmokeTest.java

```diff
@@ -661,6 +664,81 @@ public void testColStatsFileSkipping(ResourceHudiTablesInitializer.TestingTable
         assertThat(prunedSplits).isEqualTo(1);
     }
 
+    @ParameterizedTest
+    @EnumSource(
+            value = ResourceHudiTablesInitializer.TestingTable.class,
+            names = {"HUDI_TABLE_WITH_SCHEMA_CAPS_COW", "HUDI_TABLE_WITH_SCHEMA_CAPS_MOR"})
+    public void testFileSkippingWithSchemaMismatchDueToColumnCase(ResourceHudiTablesInitializer.TestingTable table)
```

Comment:
Does this test fail without the case sensitivity fix, to make sure that if the logic is broken again the test can catch it?